install_config/authentication/configuring_authentication_allow_all.adoc

```diff
@@ -0,0 +1,132 @@
+[[configuring-authentication-allow]]
+= Configuring the Allow All Identity Provider
+{product-author}
+{product-version}
+:data-uri:
+:icons:
+:experimental:
+:toc: macro
+:toc-title:
+:prewrap!:
+
+
+Setting the *AllowAllPasswordIdentityProvider* in the `*identityProviders*` stanza allows any non-empty user name and password to log in. This is the default
```

Comment:
Throughout, any place where we still have backtick+asterisk like this, while you're here might as well simplify them down to just backticks (since we updated our guidelines to no longer do the former). The single asterisks (like `*AllowAllPasswordIdentityProvider*`) could also just be backticks now too.